go.mod

```diff
@@ -7,9 +7,9 @@ toolchain go1.22.4
 require (
 	cosmossdk.io/math v1.3.0
 	github.com/avast/retry-go/v4 v4.5.1
-	github.com/babylonlabs-io/babylon v0.9.0
+	github.com/babylonlabs-io/babylon v0.9.2
```

Comment:
Do we need to update the dependency here? Seems bit like a last min change

Reply:
Nope if we don't need all the dependencies to have the latest LICENSE

Reply:
reverted all the dependency bump